Astyle.css

```diff
@@ -0,0 +1,378 @@
+  padding-left: 17px;
+  padding-right: 17px;
+  padding-bottom: 17px;
+  background: url(./assest/event.png);
+  background-size: cover;
+  text-align: left;
+  background-repeat: no-repeat;
+  font-family: 'cocogoose', sans-serif;
+}
+
+.hello {
+  margin-bottom: 0;
+  font-size: 1.7em;
+}
+
+h1 {
+  margin-top: 0;
+  font-size: 1.7em;
+}
+
+.intro .hello,
+.intro h1,
+h2,
+h3,
+h4,
+.card span {
+  font-family: 'cocogosse', sans-serif;
+  color: #ec5242;
+}
+
+.description {
+  border: 1px solid grey;
+  background-color: rgba(255, 255, 255, 0.616);
+  z-index: 10;
+  word-spacing: 2px;
+  font-size: larger;
+  margin-bottom: 26px;
+  padding: 2%;
+}
+
+.intro a {
+  color: #272a31;
+}
+
+p {
+  font-family: 'lato', sans-serif;
+  font-weight: 500;
+}
+
+.about {
+  padding: 3% 15%;
+  box-shadow: 0 3px 0 0 rgba(204, 181, 181, 0.76);
+  -webkit-box-shadow: 0 3px 0 0 rgba(204, 181, 181, 0.76);
+  -moz-box-shadow: 0 3px 0 0 rgba(204, 181, 181, 0.76);
+}
+
+.Logo {
+  display: flex;
+  gap: 6px;
+  justify-content: center;
+  align-items: center;
+  text-align: left;
+  text-decoration: none;
+  color: #272a31;
+  margin: 50px;
+  border: 1px dotted #272a31;
+}
+
+.Logo img {
+  width: 150px;
+  height: auto;
+}
+
+.Logo h2 {
+  color: #272a31;
+  width: 182px;
+  margin-left: -20px;
+}
+
+.broughtBy {
+  margin-bottom: 0;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  gap: 6px;
+  align-items: center;
+  text-decoration: none;
+  color: #272a31;
+}
+
+.about h4 {
+  font-family: 'lato', sans-serif;
+  font-size: x-large;
+  color: #272a31;
+  font-weight: 700;
+  margin-bottom: 0;
+}
+
+.footer h4 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: x-large;
+  color: #272a31;
+  font-weight: 400;
+  margin-top: 55px;
+}
+
+.imgs {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.img {
+  position: relative;
+  height: 250px;
+  width: 350px;
+  margin: 0 auto;
+}
+
+.img img {
+  width: 100%;
+  height: 100%;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  opacity: 1;
+  height: 100%;
+  width: 100%;
+  background-color: rgb(252, 86, 74, 0.6);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  color: white;
+}
+
+.overlay:hover {
+  opacity: 0.5;
+}
+
+.overlay h4 {
+  color: white;
+  margin: 0;
+}
+
+.foot {
+  margin-top: 30px;
+  text-align: left;
+  gap: 40px;
+  align-items: center;
+  padding: 8% 15%;
+  background-color: #272a31;
+  display: flex;
+  color: white;
+}
+
+.foot img {
+  margin: 0;
+  width: 100px;
+  height: auto;
+}
+
+.fbrand {
+  display: flex;
+  gap: 6px;
+  justify-content: start;
+  align-items: center;
+  width: 35%;
+  text-align: left;
+  text-decoration: none;
+  color: #272a31;
+}
+
+.fbrand img {
+  width: 70px;
+  height: auto;
+}
+
+.lfooter {
+  background-color: rgba(228, 222, 222, 0.904);
+  margin-top: 60px;
+  padding: 3% 12%;
+  display: flex;
+  gap: 50px;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly change the background color and the text color of the About Page footer section, you have to stick to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),

- Your design:

![30footer](https://user-images.githubusercontent.com/79058364/176781196-cee15322-fb05-4761-a825-079c87ecda39.png)



- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![footabou1](https://user-images.githubusercontent.com/79058364/170037105-cdec3174-540d-42d9-96a6-5390be014cae.png)

Reply:
Hi @BenMukebo,
I also wanted to match the model color,
But i couldn’t find a white logo for it …

Reply:
Is this rly something we can’t move on without ?
Its a a few keyboard taps away
If i had the logo …